### ISSUE TITLE
Generate error if someone attempts to instantiate typing alias for standard collections

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#3921]
The following will generate a runtime error, but pyright doesn't detect it.

```python
from typing import Dict
Dict({})
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.271, which I just published. It will also be included in a future release of pylance.
